app/views/devise/shared/_navbar.html.erb

```diff
@@ -0,0 +1,8 @@
+<nav>
+  <h1>Budget</h1>
+  <ul class='nav-items'>
+    <li class='nav-item'><%= link_to 'Home', groups_path %></li>
+    <li class='nav-item'><%= link_to 'Groups', groups_path %></li>
+    <li class='nav-item'><%= link_to 'Transactions', entities_path %></li>
+  </ul>
+</nav>
```

Comment:
- [ ] The application is supposed to be for  mobile devices, but when I open your page in mobile view, some part of content is gone:
<img width="517" alt="Screenshot 2022-06-03 at 22 11 16" src="https://user-images.githubusercontent.com/77038610/171922109-df189718-b8b7-4dfd-b47f-ba30bc072d8e.png">
Kindly make sure that the app renders properly in mobile screens 😉 

- [ ] I noticed that you have 2 navlinks with the same link, `home` and `groups`. Kindly delete 1